examples/binary-bitwise-expressions/binary_bitwise_expressions.bal

```diff
@@ -0,0 +1,38 @@
+import ballerina/io;
+import ballerina/lang.'int as ints;
+
+public function main() {
+    int a = 385;
+    ints:Unsigned8 b = 128;
+    // For a bitwise AND (`&`) expression, if the type of either operand is a subtype of
```

Comment:
Isn't `&` actually an operator?

Reply:
The expression itself is is a binary expression

Reply:
The spec mentions these as bitwise-and-expr, bitwise-or-expr, etc. Hence used the same.

Reply:
Then shall we first properly define what we mean by "bitwise AND expression" etc.? Because when referring to AND or `&`, what we immediately think of is the operator. The term "bitwise AND expression" sounds a bit alien.

Reply:
Any suggestions? Wanted to highlight the `&` operator along with AND. 

Do we need to "define" what a "bitwise AND expression" is? IMO, it's clear even though the terms aren't familiar.

Reply:
I'm ok with using the term bitwise AND expression. But using `&` within parenthesis like here is kinda misleading. It's why I assumed you meant the `&` operator when I first saw this.

Reply:
No need for a lengthy definition. Can expand the description you've given in the .description file to make it clear what a bitwise AND expression is

Reply:
Any suggestions?

Reply:
Can you recheck after the changes btw.

I've said 

`For a bitwise "AND" (`&`) expression,`

emphasizing "AND" and the having the operator within parentheses. Is it still not clear?